org.eclipse.lemminx/src/test/java/org/eclipse/lemminx/dom/DOMParserTest.java

```diff
@@ -107,9 +109,9 @@ public void insideEndTag() {
 		html.addChild(meta);
 
 		assertDocument("<html></meta></html>", html);
-		Assert.assertFalse(meta.hasStartTag());
-		Assert.assertTrue(meta.hasEndTag());
-		Assert.assertEquals(meta.getEndTagOpenOffset(), 6); // |</meta>
+		assertFalse(meta.hasStartTag());
+		assertTrue(meta.hasEndTag());
+		assertEquals(meta.getEndTagOpenOffset(), 6); // |</meta>

```

Comment:
Please swap parameters so that it becomes `assertEquals(6, meta.getEndTagOpenOffset());`